eclass/kde5-meta-pkg.eclass

```diff
@@ -14,6 +14,21 @@ _KDE5_META_PKG_ECLASS=1
 
 inherit kde5-functions
 
+# @ECLASS-VARIABLE: KDE_L10N_DEPS
+# @DESCRIPTION:
+# An array of ${PV} which lists kde-apps/ L10N packages to add to RDEPEND.
+# Currently known L10N packages are: kde-l10n, kde4-l10n, kdepim-l10n, ktp-l10n
+# If non-empty, add nls to IUSE to control these dependencies.
+if [[ -n ${KDE_L10N_DEPS} ]]; then
+	IUSE+=" nls"
+	local _L10N_MINIMAL _L10N_DEP
```

Comment:
It is global scope, so 'local' should not be used. This line can be dropped. 'unset' is already properly used below.